extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -348,7 +348,8 @@ public function testGetContext(
 			'opts' => [
 				'pageType' => 'all_ads',
 				'showAds' => true,
-				'delayBtf' => true
+				'delayBtf' => true,
+				'sourcePointRecovery' => false
 			],
 			'targeting' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "targeting" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)